### ISSUE TITLE
No pose supervision?

### ISSUE THREAD

**Issue by csxieyc97** [feixue94/imp-release#15]
It seems that I didn't find the any estimation or supervision of pose of the training phase in code.

**Comment by EpipolarWagner** [comment]
Yes similar for me. I couldn't find is. Where is the implementation of the weighted 8-point algorithm.
cv2.findEssentialMat, uses ransac and is none differentiable.
